### PR TITLE
Add sanity tests for CNAO alerts

### PR DESCRIPTION
**Pull request by assafad**
This PR adds new test cases for CNAO alert definitions. We expect to see `summary`, `severity`, `kubernetes_operator_part_of` and `kubernetes_operator_component` for each alert. Those fields must not be empty and some must have specific values.

Signed-off-by: assafad <aadmi@redhat.com>

```release-note
None
```

### CODE REVIEW

**Comment by assafad** [comment]
Hi @RamLavi, @sradco. Can you please review this PR?

File: test/e2e/monitoring/rules_test.go [hunk 1]
```diff
@@ -32,32 +33,76 @@ var _ = Context("Prometheus Rules", func() {
 			CheckConfigCondition(gvk, ConditionAvailable, ConditionTrue, 15*time.Minute, CheckDoNotRepeat)
 		})
 
-		Context("CNAO rules", func() {
-			It("should have available runbook URLs", func() {
-				prometheusRule := monitoringv1.PrometheusRule{}
+		AfterEach(func() {
```
Comment (RamLavi) [inline test/e2e/monitoring/rules_test.go:36]: moving the After each is ok by me, but please move it to a separate PR. this makes it hard to review the actual changes.

**Review by RamLavi — CHANGES_REQUESTED** [review]
Please ping me after the change is done so I could properly review

File: test/e2e/monitoring/rules_test.go (hunk 1 above)
Comment (assafad) [reply test/e2e/monitoring/rules_test.go]: Thanks, removed from this PR.
